server/src/main/java/org/opensearch/search/query/QueryPhaseSearcherWrapper.java

```diff
@@ -58,9 +58,10 @@ public boolean searchWith(
         boolean hasTimeout
     ) throws IOException {
         if (searchContext.shouldUseConcurrentSearch()) {
-            LOGGER.info("Using concurrent search over segments (experimental)");
+            LOGGER.debug("Using concurrent search over segments (experimental)");
```

Comment:
I added it to `debug` instead of `info` to see if there are cases where request is supposed to take non concurrent path and concurrent aggregation processor is used instead. However, I plan to remove it as part of GA readiness and when metrics will be available. So keeping it for now.